manim/__main__.py

```diff
@@ -1,5 +1,6 @@
 import sys
 import click
+import requests
```

Comment:
Is requests dependency added?

Reply:
This would need to be added to `pyproject.toml`

Reply:
Fairly sure manim already has a dependency on requests through one of our other dependencies

Reply:
> Fairly sure manim already has a dependency on requests through one of our other dependencies

anyhow we need to add it though.